hw_diag/diagnostics/env_var_diagnostics.py

```diff
@@ -19,6 +19,9 @@
 }, {
     'ENV_VAR': 'VARIANT',
     'DIAGNOSTIC_KEY': 'VA'
+}, {
+    'ENV_VAR': 'FIRMWARE_SHORT_HASH',
```

Comment:
I think part of the confusion was that this array structure is hard to understand (I wrote it originally, whoops). You have to dig through the code to discover that the `ENV_VAR` is used as the friendly_key and that `DIAGNOSTIC_KEY` is the key. 

Not worth creating a ticket just to refactor, but the next time one of us touches this code, it might be worth refactoring to something like:

```
[
...
  {
    'ENV_VAR': 'FIRMWARE_SHORT_HASH',
    'DIAGNOSTIC_KEY': 'firmware_short_hash
    'DIAGNOSTIC_FRIENDLY_KEY': 'firmware_short_hash'
  }
]
```

@kashifpk taking it a step towards https://github.com/NebraLtd/hm-pyhelper/issues/104, it could be something like:

```
from hm_pyhelper.constants import DIAGNOSTICS_KEYS

[
...
  {
    'ENV_VAR': 'FIRMWARE_SHORT_HASH',
    'DIAGNOSTIC_KEY': DIAGNOSTICS_KEYS.firmware_short_hash,
    'DIAGNOSTIC_FRIENDLY_KEY': DIAGNOSTICS_KEYS.firmware_short_hash
  }
]
```

What do you think?

Reply:
@marvinmarnold - yes, the only thing I'd change would be to have the constants in all upper case. so something like

```python
from hm_pyhelper.constants.diagnostics import FIRMWARE_SHORT_HASH_KEY
```

Already using similar logic in https://github.com/NebraLtd/Hotspot-Production-Tool/blob/master/app/src/hpt/constants.py

These would get moved to https://github.com/NebraLtd/hm-pyhelper/issues/104 once we tackle that issue.